runner/runner.go

```diff
@@ -995,6 +995,16 @@ retry:
 	if scanopts.Unsafe {
 		req.Header.Add("Connection", "close")
 	}
+	var faviconResp *httpx.Response
+	if scanopts.Favicon {
+		var favErr error
+		favURL := fmt.Sprintf("%s://%s/favicon.ico", protocol, URL.Host)
+		favreq, _ := hp.NewRequest(method, favURL)
+		faviconResp, favErr = hp.Do(favreq, httpx.UnsafeOptions{})
+		if favErr != nil {
```

Comment:
Favicon errors are not fatal. Consider only printing a warning in verbose/debug mode and continuing the execution

Reply:
Done